.env.test

```diff
@@ -2,7 +2,6 @@ LMS_BASE_URL='http://localhost:18000'
 DATA_API_BASE_URL='http://localhost:8000'
 ECOMMERCE_BASE_URL='http://localhost:8000'
 LICENSE_MANAGER_BASE_URL='http://localhost:18170'
-FEATURE_FLAGS={}
```

Comment:
yeah tests should just mock a feature toggle, if they need it anyway so I think this should be fine. some of it can also go to jest.config.js or setupTests.js if we really need it global

Reply:
What works as intended? I am like 99% sure this Feature flags var _doesn't_ work as intended, which is why I just chucked it out.

Reply:
Also the test still pass with this chucked, so 🤷